AtlasLootClassic_DungeonsAndRaids/data.lua

```diff
@@ -2936,6 +2936,25 @@ data["BlackrockDepths"] = {
 				{ 4,  22317 }, -- Lefty's Brass Knuckle
 			},
 		},
+		{ -- BRRDCorenDirebrew 
+			name = AL["Coren Direbrew"],
+			npcID = 23872,
```

Comment:
add:
`level: 73,`

Reply:
Didn't included it cause Theldren (another extra boss) hadn't level but why not.